README.md

```diff
@@ -1,182 +1,88 @@
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+Still under development:
+- Optimal layout for tablet and desktop views for creating invoice
+- Single invoice view
+- Create invoice view
+- Error handling
+- Loader and Error pages
+- Tests
+- Favicon and title changing
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Screenshots
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Invoice list light theme
+![App Screenshot](https://lh3.googleusercontent.com/pw/AJFCJaWcVHKdVkmX8pXxwQZOzu3vUCQEUMe-msqREo40YhdJ9Eez6896t1KHMvMZ1SU-sLGdTPmsX3SPXsZ6Km4T01QkcLP1ee7H3dRG-ZwoWDAPTl9z6AE=w2400)
 
-### `npm run eject`
+Invoice list dark theme
+![App Screenshot](https://lh3.googleusercontent.com/pw/AJFCJaVHgBNolJ5pvr4KYITEYP2BbZA49soEo4d-jWt8h4JPSXmi_H6-WPxuH1Xu8VwMdiDKg93otToofA6nUtvJiYs5ZlCmUYAe8e_ymv4aMP8QjNOooTw=w2400)
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+Filter invoice list by drafts light theme
+![App Screenshot](https://lh3.googleusercontent.com/pw/AJFCJaWGWLGRqA7wlPVNI7umYi7skX3yH_Wdse9rTFPJeeGYtL_BtHoqQwpezkFQVMTI_okG1WHckvN5FvLU06vNZNx1Jw_2hH2Ap5Pli_hRbjmYMhMWGZU=w2400)
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+Filter invoice list by pendings dark theme
+![App Screenshot](https://lh3.googleusercontent.com/pw/AJFCJaVsybM7LCJ21R889bhNTgZZrAABk04hHzik4oGw2T6SRfEbWrFhMudckkmHJy5ZFpRjoBacBEd1T1AwOuE8cksoEAM2RO71_QaqsVB9a_vooSddK8A=w2400)
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Create invoice
+![App Screenshot](https://lh3.googleusercontent.com/pw/AJFCJaUpup2oQJ7Y2DTHvlT6dceITMrfWIgpEO8P0ePmelandTubT6ipP_G-SSVg75XAMyn7XOjQblzhgpHfxw3Xx7wHJng8SIvmFMAIv0eToWPawib27aU=w2400)
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
 
-## Learn More
+## Tech Stack
```

Comment:
I think we could move the Tech Stack section up, eg. before screenshots?

Reply:
no problemo, it's also in about section when you enter the repo